templates/test/infopanel.html.ep

```diff
@@ -85,6 +85,13 @@
                         <span title="VNC port: <%= 90 + $worker->instance %>">
                             %= link_to $worker->to_string => url_for('admin_worker_show', worker_id => $worker->id)
                         </span>
+                    % if ($job->state eq 'waiting') {
+                        <div>
+                            <span title="Developer mode available">
+                                System is waiting for developer, connect to <b><%= $worker->host %> at port <%= 90 + $worker->instance %> with Shared mode (otherwise the job might be cancelled)</b>
```

Comment:
I wonder if making this a list is better than squeezing a lot of test inside.

Reply:
I'd like to move these instructions here: https://github.com/os-autoinst/openQA/pull/1691

Otherwise, just make the div `class="developer-mode-element" data-visible-on="isPaused"` and get rid of check for waiting. This way it will automatically be shown and hidden correctly when #1691 is merged (without further JavaScript code!).